src/Data/Bookmark.elm

```diff
@@ -30,21 +32,33 @@ type alias Bookmark =
     { created : Posix
     , name : String
     , query : Query
+    , subScope : Maybe Scope
```

Comment:
Why not `scope` directly?

Reply:
Because it helps disambiguating the scope of the page and the scope of the bookmark, which are different things. A page scoped with "objects" might list veli bookmarks.